src/content/guides/getting-started.md

```diff
@@ -230,14 +243,28 @@ Given it's not particularly fun to run a local copy of webpack from the CLI, we
 
 __package.json__
 
-``` json
+``` diff
 {
-  ...
+  "name": "webpack-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
   "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Eliminate this line and description, keywords and maybe author or put something like `webpack support team`, to avoid noise and unnecessary lines in the example.

Reply:
This is the default `package.json` file generated by `npm init -y`, I would rather keep it so people feel confident that they're doing the right thing.

I know some of this is a bit verbose to us, but a lot of beginners have expressed their desire for more verbose guides as they feel webpack (and FE dev in general) can be a bit daunting.

I'm okay with making this shorter again, but I don't think it hurts to show the file here.